app/js/utils/helpers.js

```diff
@@ -55,6 +55,12 @@ export const filterThrough = (filters, data) => {
     const filteredData = matchSorter(originalData, inputValue, { keys: ['concept.display'] });
     originalData = filteredData;
   }
+
+  if (filters.testStatusField !== "All") {
```

Comment:
I'm not sure if we would need to add filters since the urgency fields themselves don't got filters either

Reply:
Dave requested for a filter on the ticket.